homeassistant/components/deconz/lock.py

```diff
@@ -0,0 +1,62 @@
+"""Support for deCONZ locks."""
+from homeassistant.components.lock import DOMAIN, LockEntity
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .const import LOCKS, NEW_LIGHT
+from .deconz_device import DeconzDevice
+from .gateway import get_gateway_from_config_entry
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Old way of setting up deCONZ platforms."""
+
+
```

Comment:
There is no need for this. Home Assistant fails gracefully with a warning if this method is used.
```suggestion
```

Reply:
Awesome, I will clean this up in the older platforms as well